bin/strata-cli/src/cmd/change_pwd.rs

```diff
@@ -12,6 +12,14 @@ pub struct ChangePwdArgs {}
 pub async fn change_pwd(_args: ChangePwdArgs, seed: Seed, persister: impl EncryptedSeedPersister) {
     let term = Term::stdout();
     let mut new_pw = Password::read(true).unwrap();
+    let entropy = new_pw.entropy();
+    let _ = term.write_line(
+        format!(
+            "Password strength (estimated crack time): {}",
```

Comment:
What about?
```suggestion
            "Password strength (estimated time needed by an attacker): {}",
```

Reply:
I think it's neat that the library can do time estimates, but I'm not sure it's particularly useful for the end user. Is a decade sufficiently strong? A thousand years? It's not really actionable.

My recommendation is instead to use the [score](https://docs.rs/zxcvbn/latest/zxcvbn/struct.Entropy.html#method.score) provided by the library (but without showing it to the user directly). If the score is at least 3 (sufficiently strong), don't provide any feedback. If the score is below that threshold, warn the user and consider providing [feedback](https://docs.rs/zxcvbn/latest/zxcvbn/struct.Entropy.html#method.feedback).

Reply:
Thanks for the suggestions, totally agreed.
I've implemented in b3709c146ae0ae77ffdc65a3e6263a23be9ac469

Reply:
PS: Screenshots updated